controllers/argocd/notifications/constants.go

```diff
@@ -0,0 +1,27 @@
+package notifications
+
+// Values
+const (
+	// ArgoCDNotificationsControllerComponent is the name of the Notifications controller control plane component
+	ArgoCDNotificationsControllerComponent = "notifications-controller"
+	NotificationsSecretName                = "argocd-notifications-secret"
+	NotificationsConfigMapName             = "argocd-notifications-cm"
+	DeploymentKind                         = "Deployment"
+	RoleKind                               = "Role"
+	RoleBindingKind                        = "RoleBinding"
+	ConfigMapKind                          = "ConfigMap"
+	SecretKind                             = "Secret"
+	ServiceAccountKind                     = "ServiceAccount"
+	TLSCerts                               = "tls-certs"
+	ArgoCDRepoServerTLS                    = "argocd-repo-server-tls"
+	CapabilityDropAll                      = "ALL"
+	VolumeMountPathTLS                     = "/app/config/tls"
+	VolumeMountPathRepoServerTLS           = "/app/config/reposerver/tls"
+	WorkingDirApp                          = "/app"
+	ImageUpgradedLabel                     = "image.upgraded"
+	TimeFormatMST                          = "01022006-150406-MST"
+	APIVersionV1                           = "v1"
+	APIVersionAppsV1                       = "apps/v1"
+	APIVersionRbacV1                       = "rbac.authorization.k8s.io/v1"
+	APIGroupRbacV1                         = "rbac.authorization.k8s.io"
```

Comment:
Some of these have been extracted to the `common` folder
other non-notifications specific constants should also be treated the same 

I think we should create an API and resources section in `common.values.go` to add these strings there and move them out of this package